change/@fluentui-react-0508f92f-47d1-484e-920e-866370166b49.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Used overflowWrap",
```

This goes in the release notes, so it's good if it's a little more descriptive.
```suggestion
  "comment": "TeachingBubble: Add overflow-wrap setting to header",
```